internal/services/network/local_network_gateway_resource_test.go

```diff
@@ -181,7 +181,7 @@ func TestAccLocalNetworkGateway_updateAddressSpace(t *testing.T) {
 
 	data.ResourceTest(t, r, []acceptance.TestStep{
 		{
-			Config: r.noneAddressSpace(data),
+			Config: r.singleAddressSpace(data),
```

Comment:
since the Update function is being tested above, presumably we should also test this scenario at the end too?